ghost/i18n/locales/ca/comments.json

```diff
@@ -3,69 +3,69 @@
     "Anonymous": "Anònim",
-    "Are you sure?": "",
+    "Are you sure?": "N'estàs segur?",
     "Become a member of {{publication}} to start commenting.": "Fes-te membre de {{publication}} per començar a comentar.",
     "Become a paid member of {{publication}} to start commenting.": "Fes-te membre de pagament de {{publication}} per començar a comentar",
     "Cancel": "Cancel·lar",
     "Comment": "Comentari",
     "Complete your profile": "Completa el teu perfil",
     "Delete": "Esborrar",
-    "Deleted": "",
+    "Deleted": "Esborrat",
     "Deleted member": "Membre esborrat",
-    "Deleting": "",
+    "Deleting": "Esborrant",
     "Discussion": "Debat",
     "Edit": "Editar",
     "Edit this comment": "Editar aquest comentari",
     "edited": "editat",
     "Enter your name": "Escriu el teu nom",
-    "Expertise": "Conéixement",
+    "Expertise": "Expertesa",
     "Founder @ Acme Inc": "Fundador @ Acme Inc",
     "Full-time parent": "Pare a temps complert",
     "Head of Marketing at Acme, Inc": "Cap de màrqueting a Acme, Inc",
-    "Hidden for members": "",
-    "Hide": "Amaga",
-    "Hide comment": "Amaga el comentari",
+    "Hidden for members": "Ocult pels membres",
+    "Hide": "Oculta",
+    "Hide comment": "Oculta el comentari",
     "Jamie Larson": "Jamie Larson",
     "Join the discussion": "Uneix-te al debat",
     "Just now": "Just ara",
-    "Local resident": "Resiudent local",
+    "Local resident": "Resident local",
     "Member discussion": "Debat de membres",
     "Name": "Nom",
     "Neurosurgeon": "Neurocirurgià",
-    "Once deleted, this comment can’t be recovered.": "",
+    "Once deleted, this comment can’t be recovered.": "Un cop esborrat, aquest comentari no es pot recuperar",
     "One hour ago": "Fa una hora",
     "One min ago": "Fa un minut",
-    "removed": "",
-    "Replied to": "",
+    "removed": "Esborrat",
+    "Replied to": "Respost a",
     "Reply": "Respondre",
-    "Reply to": "",
+    "Reply to": "Respondre a",
     "Reply to comment": "Respondre al comentari",
     "Report": "Reportar",
     "Report comment": "Reportar comentari",
     "Report this comment?": "Reportar aquest comentari?",
-    "Save": "Desa",
+    "Save": "Desar",
     "Sending": "Enviant",
-    "Sent": "Enviar",
+    "Sent": "Enviat",
     "Show": "Mostra",
-    "Show {{amount}} more replies": "Mostra {{amount}} respostes més",
-    "Show {{amount}} previous comments": "Mostra {{amount}} comentaris previs",
-    "Show 1 more reply": "Mostra 1 resposta més",
-    "Show 1 previous comment": "Mostra 1 comentari previ",
-    "Show comment": "Mostra el comentari",
-    "Sign in": "Inici de sessió",
+    "Show {{amount}} more replies": "Mostrar {{amount}} respostes més",
+    "Show {{amount}} previous comments": "Mostrar {{amount}} comentaris previs",
+    "Show 1 more reply": "Mostrar 1 resposta més",
+    "Show 1 previous comment": "Mostrar 1 comentari previ",
+    "Show comment": "Mostrar comentari",
+    "Sign in": "Iniciar sessió",
```

Comment:
another question about imperative vs infinitive, if you could please confirm.  Context:  These are a labels for a link or a button.

Reply:
I'll change them to imperative as per Softcatala's instructions: https://www.softcatala.org/forum/viewtopic.php?f=4&t=269 although Imperative might be preferred for certain generic actions like seach or sign in. We will keep imperative for specific actions like "show 1 more reply".